cmd/scollector/collectors/snmp_ifaces.go

```diff
@@ -125,6 +134,7 @@ func c_snmp_ifaces(community, host string) (opentsdb.MultiDataPoint, error) {
 			}
 			Add(&md, switchInterfaceMetric(sA.metric, ifNames[k], ifTypes[k]), v, tags, sA.rate, sA.unit, sA.desc)
 			metadata.AddMeta("", tags, "alias", ifAliases[k], false)
+			metadata.AddMeta("", tags, "mac", ifPhysAddresses[k], false)
```

Comment:
should we take this opportunity to reduce the set of tags we are adding metadata for? Do we really need iname and direction in the metadata tagset?